src/main/frontend/pipeline-console-view/pipeline-console/main/pipeline-console.scss

```diff
@@ -2,12 +2,12 @@
   // can be removed and `--header-height` used directly when jenkins baseline is 2.507
   --pgv-header-height: var(--header-height, 45.5938px);
   --ansi-0: var(--black, #333);
-  --ansi-1: var(--danger, #cc0003);
-  --ansi-2: var(--success, #138347);
-  --ansi-3: var(--warning, #ea6b19);
-  --ansi-4: var(--primary, #0b6aa2);
-  --ansi-5: var(--unknown-color, #bd0fe1);
-  --ansi-6: var(--paused-color, #24b0d5);
+  --ansi-1: var(--red, oklch(70% 0.15 27));
```

Comment:
Looks good to me! 

I don't think we need these fallback colors however?

Reply:
Well, maybe just for extra robustness. The variables provided by the Jenkins core are quite far away from this place. While it's unlikely that they'll ever break backward compatibility, anything could happen in the future.

In my opinion, they don't harm. But if you think they are just cluttering the code, I'll remove them, no questions asked.

Reply:
Let's remove them - I'm thinking they're just a holdover from before Jenkins had a set palette of colours. 

Thanks!

Reply:
they are part of the public interface at least the variable names so they won't change.

values can change but the fallbacks here will never be used.

Only time we normally use a fallback is if we haven't increased the minimum version of Jenkins core required but we want to use a new variable.